.github/workflows/ci.yaml

```
@@ -0,0 +1,27 @@
+name: Continuous Integration
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ v1-mvp ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [18.x]
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

```suggestion
        uses: actions/checkout@v3
```
Latest version is 3 as per [here](https://github.com/marketplace/actions/checkout)

---

Cool good catch!